deployment/environment/crib/helpers.go

```diff
@@ -0,0 +1,92 @@
+package crib
+
+import (
+	"context"
+	"sync"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	gethtypes "github.com/ethereum/go-ethereum/core/types"
+	chainsel "github.com/smartcontractkit/chain-selectors"
+
+	"math/big"
+
+	"github.com/smartcontractkit/chainlink-testing-framework/lib/utils/conversions"
+	"github.com/smartcontractkit/chainlink/deployment"
+	"github.com/smartcontractkit/chainlink/deployment/environment/devenv"
+	"github.com/smartcontractkit/chainlink/v2/core/logger"
+)
+
+var wg = &sync.WaitGroup{}
+
+func distributeFunds(lggr logger.Logger, nodeInfo []devenv.Node, env deployment.Environment) {
+	transmittersStr := make([]common.Address, 0)
+	fundingAmount := big.NewInt(500_000_000_000_000_000) // 0.5 ETH
```

Comment:
Easily understandable
```suggestion
	fundingAmount := new(big.Int).Mul(UBigInt(5), UBigInt(1e17))
	minThreshold := new(big.Int).Mul(UBigInt(5), UBigInt(1e16))
```